orm/main.go

```diff
@@ -82,6 +116,16 @@ func createOrmServer() *martini.ClassicMartini {
 		}
 		writeChallenge(rs, challenge)
 	})
+
+	m.Delete("/challenge/:id", binding.MultipartForm(ChallengeForm{}), func(rs http.ResponseWriter, rq *http.Request, f ChallengeForm, params martini.Params) {
+		id := params["id"]
+		err := orm.DeleteChallenge(id)
+		if err != nil {
+			utils.WriteResponse(rs, http.StatusInternalServerError, "Database error trying to find challenge with id "+string(id)+" to update", err)
+			return
+		}
+		utils.WriteResponse(rs, http.StatusNoContent, "Successfully deleted", nil)
```

Comment:
You can't say `204 No Content` and have content on the body at the same time, that's against federal law 👮